.github/PULL_REQUEST_TEMPLATE/bug_fix_template.md

```diff
@@ -0,0 +1,51 @@
+# Bug Fix 
+**Name:** 
+
+**Issue Reference:**
+
+
+## Modified
+
+**Class/Function Name:** *ClassName* 
+
+**module:** *python_filename.py*
+
+**package:** */directory_name*
+
+*Please provide any other information on what is modified to fix this bug*
+
+## Scope
+- Which bug does this address?
+- *use point form or check boxs*
+- What does this bug fix not address? 
+
+### Requirements Checklist
+- [ ] PEP 8/flake8 check
+- [ ] current unit tests -> **pass**
+- [ ] Update code documentation (if applicable)
+- [ ] Update readthedocs documentation (if applicable)
+- [ ] merge to `develop` or `master`
+
+### Merging to Master
+*If applicable please reason why this bug fix should be merged to `master`?*
```

Comment:
To prevent heated discussions, I would recommend adding a list of acceptable reasons here.

Reply:
done